python/distant_vfx/utils.py

```diff
@@ -16,16 +16,16 @@ def format_html_table(rows):
         header_row = rows.pop(0)
         header_html = ''
         for col in header_row:
-            header_html += f'<th>{col}</th>'
+            header_html += '<th>{col}</th>'.format(col=col)
```

Comment:
Not pictured here, might be worth looking into `jinja2` if you are doing templating.

Just might be nicer to write and read

Reply:
Notes, thanks! I was just starting to mess around with this, haven't gotten very far. I'm not even sure we'll end up using html for the emails, but if we do I'll look into `jinja2`.